Request-464366 (#445)
newrelic/helm-charts: charts/newrelic-infra-operator/README.md

| `nodeSelector`                                                                  | Node label to use for scheduling.                                                                                                                                                                                                                                                                                                           | `{}`                                                                                                 |
| `timeoutSeconds`                                                                | Seconds to wait for a webhook to respond. The timeout value must be between 1 and 30 seconds.                                                                                                                                                                                                                                               | `30`                                                                                                 |

> | `tolerations`                                                                   | List of node taints to tolerate (requires Kubernetes >= 1.6)                                                                                                                                                                                                                                                                                | `See Tolerations below`                                                                              |

Reviewer: ```suggestion
| `tolerations`                                                                   | List of node taints to tolerate (requires Kubernetes >= 1.6)                                                                                                                                                                                                                                                                                | Tolerations for the operator deployment and jobs. See `Tolerations` below.                                                                              |
```

Author: Modified the file with the suggestions

Author: Lint might fail again, I need to disable few stuff on my IDE so files don't get auto formated on save.

If you have by anychance a `.editorconfig` files with your standards would be great.

Thanks,

    cpu: 150m
```

## Tolerations

> The default set of relations assigned to our job is shown below:

Reviewer: ```suggestion
The default set of relations assigned to the operator itself and the related jobs are shown below:
```

Author: modify the file with the suggestion

```yaml
- operator: "Exists"
  effect: "NoSchedule"
- operator: "Exists"
  effect: "NoExecute"
```

> 

Reviewer: ```suggestion

Please note that these tolerations are applied only to the operator and the certificate-related jobs themselves, and not to any pod or container injected by it.

```

Author: modify the file with the suggestion
